istioctl/cmd/root.go

```diff
@@ -199,36 +199,36 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(precheck.Cmd(ctx))
 	experimentalCmd.AddCommand(proxyconfig.StatsConfigCmd(ctx))
 	experimentalCmd.AddCommand(checkinject.Cmd(ctx))
-	rootCmd.AddCommand(waypoint.Cmd(ctx))
-	rootCmd.AddCommand(ztunnelconfig.ZtunnelConfig(ctx))
+	rootCmd.AddCommand(setNotSupportedCmd(waypoint.Cmd(ctx).Name()))
+	rootCmd.AddCommand(setNotSupportedCmd(ztunnelconfig.ZtunnelConfig(ctx).Name()))
 
 	analyzeCmd := analyze.Analyze(ctx)
 	hideInheritedFlags(analyzeCmd, cli.FlagIstioNamespace)
 	rootCmd.AddCommand(analyzeCmd)
 
 	dashboardCmd := dashboard.Dashboard(ctx)
 	hideInheritedFlags(dashboardCmd, cli.FlagNamespace, cli.FlagIstioNamespace)
-	rootCmd.AddCommand(dashboardCmd)
+	rootCmd.AddCommand(setNotSupportedCmd(dashboardCmd.Name()))
 
 	manifestCmd := mesh.ManifestCmd(ctx)
 	hideInheritedFlags(manifestCmd, cli.FlagNamespace, cli.FlagIstioNamespace, FlagCharts)
 	rootCmd.AddCommand(manifestCmd)
 
 	installCmd := mesh.InstallCmd(ctx)
 	hideInheritedFlags(installCmd, cli.FlagNamespace, cli.FlagIstioNamespace, FlagCharts)
-	rootCmd.AddCommand(installCmd)
+	rootCmd.AddCommand(setNotSupportedCmd(dashboardCmd.Name()))
```

Comment:
ah, this is the reason. This should be 
```suggestion
	rootCmd.AddCommand(setNotSupportedCmd(installCmd.Name()))
```

Reply:
Good catch, thanks !

Reply:
Done